self-tests.yml

```diff
@@ -721,6 +728,12 @@ tasks:
           permissions: public-read
           display_name: swagger.json (latest)
           patchable: false
+  - name: write-downstream-expansions-pine
+    commands: 
+      - func: write-downstream-expansions-for-pine
```

Comment:
the name and the function name being near identical seems confusing to me. maybe we should just inline the command and call this task `write-downstream-expansions-for-pine`